typescript/infra/scripts/utils.ts

```diff
@@ -171,6 +180,7 @@ export function getKeyForRole(
   role: Role,
   index?: number,
 ): CloudAgentKey {
+  debugLog(`Get key for ${role} role`);
```

Comment:
nit: not important but elsewhere we say stuff like "Getting ..." etc instead of "Get ...", would be good to be consistent throughout here / this PR